spec/requests/moderations_spec.rb

```diff
@@ -135,4 +135,54 @@
       expect(response.body).to include "circle centered-icon adjustment-icon subtract"
     end
   end
+
+  describe "/mod" do
+    # rubocop:disable Layout/LineLength
+    coc_guides_copy = 'Check out our <a href="https://dev.to/code-of-conduct">Code of Conduct</a> and read through our <a href="https://dev.to/community-moderation">Trusted User Guide</a> and <a href="https://dev.to/tag-moderation">Tag Moderation Guide</a>.'
+    # rubocop:enable Layout/LineLength
+    become_mod_copy = "If you'd like to assist us as a trusted user or tag mod"
+    logged_out_copy = "P.S. You are not currently signed in."
```

Comment:
Can we use Rspec's `let` [helper](https://relishapp.com/rspec/rspec-core/v/2-6/docs/helper-methods/let-and-let) here instead of using variables directly?

Same for line 152 and 178, and anywhere else in the file :)